src/optimisation/Optimisation.jl

```diff
@@ -1,324 +1,327 @@
 module Optimisation
 
 using ..Turing
+using ..OptimisationCore: ModeResult, MLE, MAP, OptimLogDensity, OptimizationContext,
+    ModeEstimator, variable_names
+using NamedArrays: NamedArray
+using Optimization
+using OptimizationOptimJL: LBFGS, IPNewton
+using DocStringExtensions: TYPEDFIELDS
```

Comment:
```suggestion
```

See comment below.

Reply:
Now qualifying these references, and importing them as e.g. `using DocStringExtension: DocStringExtension`.